tutorials/howtos/forms.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Building a Contact Form with Apostrophe"
```

Comment:
I would change this to "Building custom forms with Apostrophe".

Reply:
Yeah, I was thinking this needed a more generalized title.